en/micro-integrator/docs/references/connectors/file-connector/file-connector-example.md

```diff
@@ -0,0 +1,146 @@
+            </fileconnector.create>
+            <respond/>
+        </inSequence>
+        <outSequence/>
+        <faultSequence/>
+    </resource>
+    <resource methods="POST" uri-template="/read">
+        <inSequence>
+            <property expression="json-eval($.source)" name="source" scope="default" type="STRING"/>
+            <fileconnector.isFileExist>
+                <source>{$ctx:source}</source>
+            </fileconnector.isFileExist>
+            <property expression="json-eval($.fileExist)" name="response" scope="default" type="STRING"/>
+            <log level="custom">
+                <property expression="get-property('response')" name="responselog"/>
+            </log>
+            <switch source="get-property('response')">
+                <case regex="true">
+                    <fileconnector.read>
+                        <source>{$ctx:source}</source>
+                    </fileconnector.read>
+                    <respond/>
+                </case>
+                <default>
+                    <log>
+                        <property name="notext" value="&quot;File does not exist&quot;"/>
+                    </log>
+                </default>
+            </switch>
+            <drop/>
+        </inSequence>
+        <outSequence/>
+        <faultSequence/>
+    </resource>
+</api>
+
+```
+
+4. Follow these steps to [export the artifacts](../exporting-artifacts.md).
+
+## Deployment
+### Deploying on WSO2 Enterprise Integrator 7
+
+You can copy the carbon application to the <PRODUCT-HOME>/repository/deployment/server/carbonapps folder and start the server. 
+Micro Integrator will be started and the composite application will be deployed. You can further refer to the application deployed through the CLI tool. Make sure you first export the PATH as below.
+
+    ```
+    $ export PATH=/path/to/mi/cli/directory/bin:$PATH\
+    ```
+
+1. Log in to Micro Integrator using the following command.
+
+    ```
+    ./mi remote login
+    ```
+
+2. Provide default credentials admin for both username and password.
+3. In order to view the APIs deployed, execute the following command.
+
+    ```
+    ./mi api show
+    ```
+
+### Deploying on WSO2 Enterprise Integrator 6
+
+1. You can copy the carbon application to the <PRODUCT-HOME>/repository/deployment/server/carbonapps folder and start the server. 
+
+2. WSO2 EI server starts and you can login to the Management Console https://localhost:9443/carbon/ URL. Provide login credentials. The default credentials will be admin/admin. 
+
+3. You can see that the API is deployed under the API section. 
+
+## Testing
+
+### File Create Operation
+
+1. Create a file called data.json with the following payload. 
+```
+{
+    "source":"/Users/IsuruUyanage/Desktop/RnD/2020/Q1/ConnectorVerification/create.txt",
+    "inputContent": "This is a test file"
+}
+```
+2. Invoke the API as below using curl. It can be downloaded from [Releases and Downloads](https://curl.haxx.se/download.html).
```

Comment:
`Curl Application` can be downloaded from [here] (https://curl.haxx.se/download.html).

Reply:
Done